manifests/stable/metallb-operator.clusterserviceversion.yaml

```diff
@@ -550,6 +550,7 @@ spec:
                   - mountPath: /tmp/k8s-webhook-server/serving-certs
                     name: cert
                     readOnly: true
+              serviceAccountName: manager-account
```

Comment:
@fedepaol This is the only new change plus the rebase with the branch.
I tested it on an OCP cluster and it works. please let's merge it if that's ok.